src/server/scripts/Northrend/AzjolNerub/ahnkahet/boss_elder_nadox.cpp

```diff
@@ -33,165 +34,165 @@ enum Spells
 enum Creatures
 {
     NPC_AHNKAHAR_SWARMER            = 30178,
-    NPC_AHNKAHAR_GUARDIAN_ENTRY     = 30176,
+    NPC_AHNKAHAR_GUARDIAN           = 30176,
+    NPC_AHNKAHAR_SWARM_EGG          = 30172,
+    NPC_AHNKAHAR_GUARDIAN_EGG       = 30173,
 };
 
 enum Events
 {
-    EVENT_CHECK_HEALTH              = 1,
-    EVENT_CHECK_HOME                = 2,
-    EVENT_PLAGUE                    = 3,
-    EVENT_BROOD_RAGE                = 4,
-    EVENT_SWARMER                   = 5,
-    EVENT_SUMMON_GUARD              = 6,
+    EVENT_CHECK_HOME                = 1,
+    EVENT_PLAGUE                    = 2,
+    EVENT_BROOD_RAGE                = 3,
+    EVENT_SWARMER                   = 4,
 };
 
 enum Yells
 {
-    SAY_AGGRO       = 0,
-    SAY_SLAY        = 1,
-    SAY_DEATH       = 2,
-    SAY_EGG_SAC     = 3,
-    EMOTE_HATCHES   = 4
+    SAY_AGGRO                       = 0,
+    SAY_SLAY                        = 1,
+    SAY_DEATH                       = 2,
+    SAY_EGG_SAC                     = 3,
+    EMOTE_HATCHES                   = 4
 };
 
 class boss_elder_nadox : public CreatureScript
 {
 public:
     boss_elder_nadox() : CreatureScript("boss_elder_nadox") { }
 
-    struct boss_elder_nadoxAI : public ScriptedAI
+    struct boss_elder_nadoxAI : public BossAI
     {
-        boss_elder_nadoxAI(Creature *c) : ScriptedAI(c), summons(me)
-        {
-            pInstance = c->GetInstanceScript();
+        boss_elder_nadoxAI(Creature* creature) : BossAI(creature, DATA_PRINCE_TALDARAM),
+            previousSwarmEgg_GUID(0),
+            guardianSummoned(false),
+            respectYourElders(true)
```

Comment:
brackets

Reply:
its initialization list, it had brackets below